wpe/src/main/cpp/Browser/CookieManager.h

```diff
@@ -0,0 +1,31 @@
+/**
+ * Copyright (C) 2022 Igalia S.L. <info@igalia.com>
+ *   Author: Jani Hautakangas <jani@igalia.com>
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE. See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public
+ * License along with this library; if not, write to the Free Software
+ * Foundation, Inc., 51 Franklin Street, Fifth Floor, Boston, MA 02110-1301 USA
+ */
+
+#pragma once
+
+class CookieManager final {
+public:
+    enum class CookieAcceptPolicy {
```

Comment:
The `enum class CookieAcceptPolicy` is only used in CookieManager.cpp as it is only called from the Java side through JNI.

It's a bit weird to have it here as it is not used anywhere else, maybe it would be better in CookieManager.cpp inside an unnamed namespace.